x-pack/plugins/cloud_security_posture/server/saved_objects/mappings.ts

```diff
@@ -60,3 +60,18 @@ export const cspBenchmarkRuleSavedObjectMapping: SavedObjectsTypeMappingDefiniti
     },
   },
 };
+
+export const cspSettingsSavedObjectMapping: SavedObjectsTypeMappingDefinition = {
+  dynamic: false,
+  properties: {
+    rules: {
+      type: 'object',
+      properties: {
+        muted: {
+          type: 'boolean',
+        },
+        // Add more properties as needed
+      },
+    },
```

Comment:
the `muted` mapping is placed under the wrong path

Reply:
WDYM?  It's under `rules`... Perhaps I misunderstood your point.

Reply:
it does not correlate with how you struct the settings

the settings looks as follows:

```ts
{
   rules: {
       rule_idX: {
           muted: true
       }
   }
}
```

while the mappings doesn't take into account the dynamic rule id and based on the current mappings the structure is:

```ts
{
   rules: {
      muted: true
   }
}
```